docs/debugging.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Blazor debugging
```

Comment:
The gerund thing again ... APEX would prefer "Debug Blazor" or something that avoids a gerund here.